bpf/kmesh/ads/include/ctx/sock_ops.h

```diff
@@ -10,6 +10,10 @@ typedef struct bpf_sock_ops ctx_buff_t;
 
 #define KMESH_PORG_CALLS sockops
 
+// tail_call map dont support pinning when shared by different bpf types, so define different name in sockops & sockconn
+#define map_of_tail_call_prog kmtailcallprogl7
+#define map_of_tail_call_ctx  kmtailcallctx_l7

```

Comment:
in enhanced mode, the macro will conflict with that defined above in bpf/kmesh/ads/include/ctx/sock_addr.h

Reply:
no, skops use sock_ops.h , and sockconn use sock_addr.h
so will not conflict